File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.py
def sort(arr):
  for i in range(len(arr) - 1):
    for j in range(i + 1, len(arr)):
      if arr[i] > arr[j]:
        arr[i], arr[j] = arr[j], arr[i]

  return arr

arr = []
for i in range(28):
  arr.append(int(input()))

an_arr = []

for i in range(1, 31):
    an_arr.append(i)

missing_numbers = []

for num in an_arr:
    if num not in arr:
        missing_numbers.append(num)

result = sort(missing_numbers)

for item in result:
  print(item)

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.c
#include <stdio.h>

void sort(int arr[], int length){
   int temp;
   for(int i = 0; i < length - 1; i++){
      for(int j = i + 1; j < length; j++){
         if(arr[i] > arr[j]){
            temp = arr[i];
            arr[i] = arr[j];
            arr[j] = temp;
         }
      }
   }
}

int main(void) {
   int arr[28];
   for(int i = 0; i < 28; i++){
      scanf("%d", &arr[i]);
   }

   sort(arr, 28);

   int index = 1;
   int print = 0;
   
   for(int i = 0; i < 28; i++){
      if(arr[i] != index){
         printf("%d\n", index);
         print += 1;
         i -= 1;
      }
      index += 1;
   }
   if (print < 2){
      printf("%d", 30);
   }
   
   return 0;
}